starfish/src/main/java/org/maxandersen/Starfish.java

```diff
@@ -0,0 +1,238 @@
+package org.maxandersen;
+
+import io.quarkus.runtime.QuarkusApplication;
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Objects;
+import java.io.File;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.dataformat.yaml.YAMLFactory;
+
+public class Starfish implements QuarkusApplication {
+  @Override
+  public int run(String... args) throws Exception {   
+    
+    
+    for(String s:args)
+    System.out.println("\nTarget : "+s);//Printing Received Arguements for Debugging
+
+    if(args[0].equalsIgnoreCase("config")){
+    //Incase user wants to configure starfish
+    editConfig();//Calling function that enables to edit configuration
+    return 10; //Incase user typed "starfish config" we only want to edit configuration
+    }
+
+    //Calling function for Fetching installed Packages 
+    //fetch_installed_packages(); //This function will print all installed packages
+
+    //Configuration identifiers
+    String clone_path="";//Holds path to  destination Where the Repository Must Be Clonned
+    String ide="";  //Holds command for IDE to open upon
+    String userHome = System.getProperty( "user.home" ); //Get User Home Directory: /home/user_name
+
+    File configFile = new File(userHome + "/starfish-config.yml"); //Loading YAML
+
+
+  
+    
+    
+    //Reading Configuration File
+    try {
+        if (!configFile.exists()) {
+            //Incase no config file exists
+            System.out.println("\nThis is the first time you're using starfish...You'll need to  configure it");
+            editConfig(); //Calling function that lets user to configure 
+        }
+        
+        System.out.println("\nLoading configurations.....\n");
```

Comment:
Use a logger.

Reply:
I just thought it would be useful to show the progress etc.
will remove it.

Reply:
currently, I've commented it out